wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,75 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
+It's hightly recommended to don't use third-party software to update drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly find the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager (Windows) (right click into windows icon and press “Device Manager”). In the Device Manager, look for the device you need drivers for and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you need drivers for.
```

Comment:
Might be better to say identify instead of find? :

The first step in installing drivers is to correctly **identify** the hardware you need drivers for.